psycopg_pool/psycopg_pool/pool_async.py

```diff
@@ -677,6 +676,23 @@ async def _connect(self, timeout: float | None = None) -> ACT:
         self._set_connection_expiry_date(conn)
         return conn
 
+    async def _resolve_conninfo(self) -> str | Any:
```

Comment:
Why returning Any?
```suggestion
    async def _resolve_conninfo(self) -> str:
```
We may declare to return `str | None`, or just to return str and return `self.conninfo or ""` at the end.